### PR TITLE
feat: post request from google api

### PR DESCRIPTION
**Pull request by ai-n0808**

### CODE REVIEW

File: src/utils/getPlacesFromGoogleByText.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    textQuery: searchText,
+  };
+
+  const query = await fetch(uri, {
+    method: "POST",
+    headers: headerForGooglePlacesApi,
+    body: JSON.stringify(foo),
+  });
+  return await query.json();
+}
```
Comment (brianaqp) [inline src/utils/getPlacesFromGoogleByText.ts:37]: Very good changes. Its clear.

**Review by kiw1meister — APPROVED** [review]
Seems good.

**Review by brianaqp — CHANGES_REQUESTED** [review]
We are not sending Google Data to the client.

A Place received by google needs to be parsed to our "Places" table, and then, pull the Photos with the place reference.

Remove the changes in the endpoint and implement it in other branch, or commit with the new changes. 

Btw, good joob getting it to work!

File: src/utils/getPlacesFromGoogleByText.ts (hunk 1 above)
Comment (kiw1meister) [reply src/utils/getPlacesFromGoogleByText.ts:37]: If I understand the code correctly, I don't think the code is sending anything to the frontend at the moment?

Comment (ai-n0808) [reply src/utils/getPlacesFromGoogleByText.ts:37]: Yes. I didn't send response to frontend yet with this code.

Comment (brianaqp) [reply src/utils/getPlacesFromGoogleByText.ts:37]: Sorry! I miss read. I guess the endpoint only work for testing the route. Thanks for clarify

Nice!

**Pull request merged**